arm64/root/start.sh

```diff
@@ -40,6 +44,33 @@ if [ "$DECONZ_VNC_MODE" != 0 ]; then
   
   # Export VNC display variable
   export DISPLAY=$DECONZ_VNC_DISPLAY
+  
+  if [ "$DECONZ_NOVNC_PORT" = 0 ]; then
+    echo "[marthoc/deconz] noVNC Disabled"
+  else
+    if [ "$DECONZ_NOVNC_PORT" -lt 6080 ]; then
+      echo "[marthoc/deconz] ERROR - NOVNC port must be 6080 or greater!"
+      exit 1
+    fi
+
+    # Assert valid SSL certificate
+    NOVNC_CERT="novnc.pem"
+    if [ -f "$NOVNC_CERT" ]; then
+      openssl x509 -noout -in "$NOVNC_CERT" -checkend 0 > /dev/null
+      if [ $? != 0 ]; then
+        echo "[marthoc/deconz] The noVNC SSL certificate has expired; generating a new certificate now."
+       	rm "$NOVNC_CERT"
```

Comment:
Just have a look at the indention ... one line is with tabs and the others without.

BTW: good point the checking the logics. You could also use `!` as negation instead of using an extra `if-then` block. I don't know what is more readable though.